docs/send-data/hosted-collectors/amazon-aws/aws-security-data-lake-source.md

```diff
@@ -0,0 +1,83 @@
+
+1. **Account ID**. Remember to copy and secure your Sumo Logic ID: ************. It will be needed in the further steps.
+2. **External ID**. The External ID is formed from your Sumo Logic region identifier. The format of your Sumo Logic account identifier is: `<SumoDeployment>: <SumoAccountId>` where:
+   * **SumoDeployment** is your Sumo Logic deployment that has to be entered in lowercase such as au, ca, de, eu, fed, in, jp, us1, or us2. To find your deployment, see [Sumo Logic API Authentication, Endpoints, and Security](docs/api/getting-started.md).
+   * **SumoAccountId** is the Organization ID shown on your Account Overview in the Sumo Logic UI. You can access it by going to **Administration** > **Account**  > **Account Overview**.
+
+
+### Step 2. Set up subscriber in Amazon Security Data Lake console
+
+Create a subscriber in Amazon Security Data Lake Console. To create a subscriber, follow the instructions below:
+1. Navigate to the Security Data Lake console.
+2. From the left navigation, click **Subscriber**.
+3. On the Subscribers page, under the **Add subscribers** tab click **Create custom subscriber**. <br/><img src={useBaseUrl('img/send-data/setup-subscriber.png')} alt="setup-subscriber" />
+4. (Optional) Provide the **Subscriber name** and **Description**.
+5. In the **Select Logs and events sources** section, you can select which resources you want to enable. There are two options:
+     * **All Logs and event sources**. All the logs and event sources will be selected if you select this option.
+     * **Specify log and event sources**. Only specific sources will be selected if you go with this option. For example, you select **Specific log and event sources**. In the next step, you will be able to view user activity, API usage in AWS services, IP traffic details, and log and event information from Amazon Route 53.
+6. In **Log and event sources**, you have these options:
+     * **CloudTrail**: View user activity and API usage in AWS services.
+     * **VPC flow logs**: View details about IP traffic to and from network interfaces in your VPC.
+     * **Route 53**: View Amazon Route 53 Resolver query logs.
+7. Access data.
+     * **S3** or **Lakeformation** is the storage option you can choose for how your subscribers will access the data.
+     :::note
+     Subscription URL will be added to the subscriber in the later step and is kept blank deliberately.
+     :::
+
+
+### Step 3. Create Amazon Security Data Lake source in Sumo Logic
+
+When you create an Amazon Security Data Lake source, you add it to a Hosted Collector. Before creating the source, identify the Hosted Collector you want to use from the existing ones or create a new Hosted Collector, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
+
+**To create an Amazon Security Data Lake source**
```

Comment:
```suggestion
**To create an Amazon Security Data Lake source:**
```

Reply:
Thanks @kimsauce. Our standard has been to _not_ include a colon for the procedure intro. If that's something we want to change I don't feeling all that strongly about it. More of a mild preference not to.